src/app/course/goal-create/goal-creation-page.component.scss

```diff
@@ -0,0 +1,63 @@
+tui-input-date{
```

Comment:
add these styles as class names no on the element.
(i.e use `.input-date` for example and add "input-date" class to the html)